README.md

```diff
@@ -40,9 +40,12 @@ $T(n)$ = 3T(n/3) + n
        = 27T(n/27) + 3n
```

Comment:
Your recurrence relation is wrong in the first place. It's not T(n) = 3T(n/3) + n, according to your code. After calling three recursive functions of the size of n/3 each, you call return sum1+sum2+sum3, which is not O(n). It takes a constant amount of time.